CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Next Version
 
 #### Added
+- Add support non-mirrored local Swift Packages in `packages`. [#808](https://github.com/yonaskolb/XcodeGen/pull/808) @freddi-kit
```

Comment:
Let's merge these 2 entries as they will be released together, with references to both PRs
`Add support for local Swift Packages`